Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
Remove unused libraries. You can modify default script in Unity system files. (Ask teacher on Discord)